leakpro/attacks/mia_attacks/lira.py

```diff
@@ -182,10 +201,11 @@ def run_attack(self:Self) -> CombinedMetricResult:
 
             if self.online:
                 in_mean = np.mean(shadow_models_logits[mask])
-                if not self.fixed_variance:
+                if len(self.shadow_models) >= 64:
```

Comment:
Seems like a magic number.

Reply:
Yes, is now commented for why it is used